### DISCUSSION TITLE
[feature request] Flutter implementation

### DISCUSSION THREAD

**Discussion by j05u3** [WalletConnect/walletconnect-monorepo#862] [Ideas]
I have seen there is a react-native implementation. But no flutter implementation yet and flutter has a strong/growing community 😁

**Comment by pedrouid** [comment]
We are aware but we are only going to be implementing for v2.0... too late for v1.0

**Reply by suleymancalik** [reply]
@pedrouid What's plan for V2 Flutter SDK?

**Comment by j05u3** [comment]
> We are aware but we are only going to be implementing for v2.0... too late for v1.0

Thanks for your answer, I understand 👍, btw is this the correct repo to post this feature request then?

**Comment by PhamNgocPhi** [comment]
Hi, Flutter has been released to version 2.5.2, so I want to know if WalletConnect already supports Flutter? Thanks. I want to implement WalletConnect for my mobile app in flutter but I can't find anything

**Comment by nicoelzer** [comment]
We would also love to use it on Flutter – any chance Flutter support is coming soon?  🥰

**Comment by maxsap** [comment]
+1 for flutter, are there plans for it?

**Comment by j05u3** [comment]
Just to have a, maybe, useful link. This is what motivated me to open this issue:

https://stackoverflow.com/questions/68214773/problems-with-dapp-development-with-flutter-and-web3/68215593?noredirect=1#comment123884591_68215593

And I think Volodymyr Bilovus already found an implementation (check comments).

[edit] Sorry, the implementation mentioned in the stackoverflow comment is for building a wallet app, not for apps connecting to it.

**Comment by nhtlquan** [comment]
+1 for flutter

**Comment by HoangLotus** [comment]
Has anyone found a solution yet?

**Comment by Abdulhaseeb101** [comment]
Please provide a flutter implementation

**Reply by lipingx** [reply]
+1

**Comment by kkpsiren** [comment]
Wen Flutter?

**Comment by jat1168** [comment]
+1

**Reply by windswrold** [reply]
+1

**Comment by jwang0408** [comment]
+1 Please.

**Comment by chadyj** [comment]
Update on Flutter.

We hear you!  

We don't have anything concrete to say right now, but it is on our roadmap.

**Reply by nileshrathore** [reply]
thanks for getting it on the roadmap. we would also love to know any estimated date on this!

**Reply by kkpsiren** [reply]
Rough estimates would be so helpful.

**Reply by suleymancalik** [reply]
@chadyj is there a published roadmap that we can see?

**Comment by ardakalayci** [comment]
Please provide a flutter implementation

**Comment by chadyj** [comment]
Some flutter news: 

There is a 3rd party WalletConnect v2 dart SDK available at https://github.com/Eucalyptus-Labs/wallet-connect-dart

**Reply by makinghappen** [reply]
@chadyj looks great, will it be also recommended as an official implementation?

**Comment by moesaid** [comment]
+1 for flutter

**Comment by jat1168** [comment]
I found another solution
https://pub.dev/packages/wallet_connect_v2

**Comment by Bravark** [comment]
Please once you release the Flutter SDK for dapps, let us know, but for now. And does the https://github.com/Eucalyptus-Labs/wallet-connect-dart and https://pub.dev/packages/wallet_connect_v2 work?

**Comment by SametSahin10** [comment]
Is this the recommended package?

https://github.com/WalletConnect/WalletConnectDartV2

**Comment by praveen-kaia** [comment]
+1 for flutter
